en-US/install.md

```diff
@@ -62,6 +62,22 @@ by running `makepkg` in the package directory
 cd tectonic
 makepkg -si
 ```
+## nix or nixOS
+You can imperatively install `tectonic` by calling `nix-env -iA nixos.tectonic` in your shell.
+You can also do so declaratively in 2 steps:
+1. create a `shell.nix` in some directory:
+```nix
```

Comment:
This line seems to break the html page: https://tectonic-typesetting.github.io/en-US/install.html. Maybe jekyll doesn't support language specification after ` ``` `.

Reply:
yikes. Thanks for catching that @rekka. I [learned of a simpler way](https://discourse.nixos.org/t/advice-for-configuring-dependencies-for-nix-users-for-oss-project-contribution/) to load a development environment in nixos so I'll PR with that fix today.